code/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -431,14 +434,52 @@
    "outputs": [],
    "source": [
     "[fastqc]\n",
-    "input: fastq, group_by = 1\n",
+    "input: fastq, group_by = is_paired_end + 1 \n",
     "output: f'{cwd}/{_input:bnn}_fastqc.html',f'{cwd}/{_input:bnn}_fastqc/fastqc_data.txt' \n",
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, cores = numThreads\n",
     "bash: expand= \"${ }\", stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout', container=container\n",
     "    fastqc ${_input} -o ${_output[0]:d}\n",
     "    unzip -o ${_output[0]:n}.zip -d ${cwd}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c2e924a4-fcc0-4f0d-91dc-f5183c09261a",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[rnaseqc_call_1, rsem_call_1, remove_adaptor]\n",
```

Comment:
Please change this line to `[rnaseqc_call_1, rsem_call_1, fastp_remove_adaptor]`